cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js

```diff
@@ -146,7 +142,12 @@ export class OpenID4VP {
    *
    * @returns {Promise<Object>} - empty promise or error if operation fails.
    */
-  async submitOIDCPresentation(kid, presentationQuery, issuer, expiry) {
+  async submitOIDCPresentation({
+    kid,
+    presentationQuery,
```

Comment:
suggested name : presentationDef
https://identity.foundation/presentation-exchange/#presentation-definition

Reply:
Updated to `presentationSubmission`, as discussed